java/pom.xml

```diff
@@ -153,6 +153,7 @@
     </dependencies>
 
     <properties>
+        <argLine>-da:ai.rapids.cudf.AssertEmptyNulls</argLine>
```

Comment:
I'm confused what the goal of this PR is.  Java assertions are off by default.  There's an explicit enable of assertions for our unit tests run, but this appears to effectively disable the newly added assertions when we run those unit tests.  That seems to make it so we never assert in practice?